src/extensions/default/brackets-show-whitespace/main.js

```diff
@@ -44,8 +44,8 @@ define(function (require, exports, module) {
     
     // --- Settings ---
     
-    var commandID          = "denniskehrig.ShowWhitespace.toggle";
-    var preferencesID      = "denniskehrig.ShowWhitespace";
+    var commandID          = "allowWhiteSpace.toggle";
```

Comment:
I'd like to see these strings get changed back to `"denniskehrig.ShowWhitespace"` and `""denniskehrig.ShowWhitespace.toggle"`, since we're using someone else's code, it seems odd to just write them out of the picture here.

These are defining the namespace for the extensions prefs and command id, and this is the only place where they get defined.  Elsewhere we'll need to use them in order to call the code.